internal/scripts/prune-preview-deploys.ts

```diff
@@ -83,6 +83,18 @@ async function deletePreviewWorkerDeployment(id: string) {
 	if (!res.ok) {
 		throw new Error('Failed to delete worker ' + JSON.stringify(await res.json()))
 	}
+
+	const queueUrl = `https://api.cloudflare.com/client/v4/accounts/${env.CLOUDFLARE_ACCOUNT_ID}/queues/tldraw-multiplayer-queue-${id}`
+	const queueRes = await fetch(queueUrl, {
+		method: 'DELETE',
+		headers: {
+			Authorization: `Bearer ${env.CLOUDFLARE_API_TOKEN}`,
+			'Content-Type': 'application/json',
+		},
+	})
+	if (!res.ok) {
```

Comment:
We don't want to error out here as some old PRs don't have queues associated with them.